keyserver/server.go

```diff
@@ -94,8 +86,7 @@ func New(conf *ServerConfig) *ConiksServer {
 	server.reloadChan = make(chan os.Signal, 1)
 	signal.Notify(server.reloadChan, syscall.SIGUSR2)
 	server.epochTimer = time.NewTimer(time.Duration(policies.EpochDeadline()) * time.Second)
-	server.state.pad = server.InitDirectory(conf.LoadedHistoryLength)
-	server.state.tbs = make(map[string]*merkletree.TemporaryBinding, Capacity)
+	server.directory = protocol.Init(policies, sk, conf.LoadedHistoryLength, conf.RegistrationCapacity)
```

Comment:
Calling ``protocol.Init()`` seems a bit awkward to me when what you're doing is initializing the server's directory. Maybe ``InitDirectory()`` would be more clear?